internal/pkg/consensus/processor.go

```diff
@@ -592,15 +625,13 @@ func (br *DefaultBlockRewarder) BlockRewardAmount() types.AttoFIL {
 }
 
 // rewardTransfer retrieves two actors from the given addresses and attempts to transfer the given value from the balance of the first's to the second.
-func rewardTransfer(ctx context.Context, fromAddr, toAddr address.Address, value types.AttoFIL, st *state.CachedTree) error {
+func rewardTransfer(ctx context.Context, fromAddr, toAddr address.Address, value types.AttoFIL, st *state.CachedTree, vms vm.StorageMap, gt *vm.GasTracker) error {
 	fromActor, err := st.GetActor(ctx, fromAddr)
 	if err != nil {
 		return errors.FaultErrorWrap(err, "could not retrieve from actor for reward transfer.")
 	}
 
-	toActor, err := st.GetOrCreateActor(ctx, toAddr, func() (*actor.Actor, error) {
-		return &actor.Actor{}, nil
-	})
+	toActor, _, err := getOrCreateActor(ctx, st, vms, toAddr, gt)
```

Comment:
Unrelated to this change, but we really should audit this behavior with a complete spec (I don't think spec has an opinion about this yet).  Right now I don't think it's possible to setup a miner with an owner that has not been created yet.  Maybe the protocol wants to be more lenient about this.  If not then this is a complicated piece of dead code among protocol compliant nodes.